cgp/node.py

```diff
@@ -76,15 +76,15 @@ def idx(self) -> int:
     def __repr__(self) -> str:
         return (
             f"{self.__class__.__name__}(idx: {self.idx}, active: {self._active}, "
-            f"arity: {self._arity}, input_nodes {self._input_nodes}"
+            f"arity: {self._arity}, address_nodes {self._addresses}"
```

Comment:
```suggestion
            f"arity: {self._arity}, addresses {self._addresses}"
```

Reply:
or "addresses of inputs", this should easily understandable as its printed to the screen